Tribler/Core/CacheDB/Notifier.py

```diff
@@ -69,6 +68,7 @@ def remove_observers(self):
             self.observers = []
 
     def notify(self, subject, changeType, obj_id, *args):
+        from Tribler.Core.Utilities.twisted_thread import callInThreadPool
```

Comment:
Can you rebase once this one is merged? https://github.com/Tribler/tribler/pull/1492